### PR TITLE
Add umask to providers/plugin.rb

### PR DESCRIPTION
**Pull request by autumnw**
To resolve the ACL issue of file /etc/rabbitmq/enabled_plugins in some harden linux images

### CODE REVIEW

**Comment by stevecstian** [comment]
@autumnw Could you elaborate more?

File: providers/user.rb [hunk 1]
```diff
@@ -85,7 +85,7 @@ def user_has_permissions?(name, vhost, perm_list = nil) # rubocop:disable all
     new_password = new_resource.password.gsub("'", "'\\\\''")
     cmd = "rabbitmqctl add_user #{new_resource.user} '#{new_password}'"
     execute "rabbitmqctl add_user #{new_resource.user}" do # ~FC009
-      sensitive true
+      #sensitive true
```
Comment (stevecstian) [inline providers/user.rb]: @autumnw Please double check README first.

File: recipes/default.rb [hunk 2]
```diff
@@ -103,7 +103,8 @@ class Chef::Resource # rubocop:disable all
     end
   else
     # We need to download the rpm
-    rpm_package = "https://www.rabbitmq.com/releases/rabbitmq-server/v#{node['rabbitmq']['version']}/rabbitmq-server-#{node['rabbitmq']['version']}-1.noarch.rpm"
+    #rpm_package = "https://www.rabbitmq.com/releases/rabbitmq-server/v#{node['rabbitmq']['version']}/rabbitmq-server-#{node['rabbitmq']['version']}-1.noarch.rpm"
+    rpm_package = "#{node['rabbitmq']['mirror']}/rabbitmq-server-#{node['rabbitmq']['version']}-1.noarch.rpm"
```
Comment (stevecstian) [inline recipes/default.rb]: @autumnw Consider to open another PR for this.

File: recipes/default.rb [hunk 3]
```diff
@@ -184,6 +185,7 @@ class Chef::Resource # rubocop:disable all
   notifies :restart, "service[#{node['rabbitmq']['service_name']}]", :immediately
 end
 
+
```
Comment (stevecstian) [inline recipes/default.rb]: @autumnw is this newline necessary?

**Comment by stevecstian** [comment]
Should be related to #182

File: providers/user.rb (hunk 1 above)
Comment (jjasghar) [inline providers/user.rb]: ditto.

File: recipes/default.rb (hunk 2 above)
Comment (jjasghar) [inline recipes/default.rb]: It's a great idea to have the ability to put in other mirrors, but to @stevecstian 's point, this is another PR.

**Comment by jjasghar** [comment]
Any updates on these suggestions/changes?

**Comment by jjasghar** [comment]
Any progress on this PR?

**Comment by cmluciano** [comment]
I pulled the umask part in with #219. The [mirror commit](https://github.com/autumnw/rabbitmq/commit/8473bdb586f9633c0067daa00b87796fd5ac7ac8) will need a bit more work.

**Comment by jjasghar** [comment]
Closing in favor for #219

**Pull request closed without merging**